components/server/src/workspace/workspace-starter.ts

```diff
@@ -936,6 +936,10 @@ export class WorkspaceStarter {
                 featureFlags.push("workspace_psi");
             }
 
+            if (await this.shouldEnableSSHCA(user, workspace.organizationId)) {
```

Comment:
I don't want to block but please discuss with @easyCZ @kylos101. Asking users to recycle workspaces seems to be more bumpy way.

Reply:
For this feature flag, I think we do not want to persist it in the DB with the workspace itself, and instead should always pull the latest value on workspace start. 

This would then allow us to toggle between both modes, and the next time a new or existing workspace is started, it would use the mode corresponding with the flag.

Reply:
Going to test how this behaves when restarting an existing workspace, after toggling the flag.